### ISSUE TITLE
Create Tooltip Component

### ISSUE THREAD

**Issue by skuhlmann** [HausDAO/daohaus-monorepo#23]
# Overview

Create a reusable basic tool tip component.
- Single warning icon
- One color
- shows message onHover 

## Details

Name: `<ToolTip />`

### Spec questions

- Does Radix have an Icon component
    - this is a good chance to define how we do icons
-  Does Radix already have a tooltip?

#### Variants

- just looks like size.

#### Props
- toolTipText
- size

## Resources/Design

https://www.figma.com/file/R1a1bwODnzCHHHw0EJALAU/DAOhaus-v3-Pattern-Library-(In-Progress)?node-id=87%3A2443

## Deliverable
- Code/PR
- Storybook deployment

**Comment by jordanlesich** [comment]
Put some love into this component to hopefully prevent some of the challenges we had with tooltips before. 

Users can choose which side it renders on, whether they want to inject JSX or strings as content, and the trigger element. But the component is also customisation.  Devs can overwrite defaults with props. 
```js
const Tooltip = ({
  content = 'Your content here',
  side = 'right',
  triggerEl = <BiErrorCircle size="1.4rem" color={COLOR.WARNING} />,
}: {
  content: string | React.ReactNode;
  side?: 'top' | 'right' | 'bottom' | 'left';
  triggerEl: string | React.ReactNode;
}) => {
  return (
    <TooltipPrimitive.Provider delayDuration={400}>
      <TooltipRoot>
        <TooltipTrigger>{triggerEl}</TooltipTrigger>
        <StyledContent side={side} sideOffset={18}>
          {content}
          <StyledArrow />
        </StyledContent>
      </TooltipRoot>
    </TooltipPrimitive.Provider>
  );
};
```

**Comment by jordanlesich** [comment]
For icons, I've been using them direct with React-Icons. So far, so good. 

As far as size goes, what part of size are we controlling? I just have it wrapping whatever gets input for ``content``, then it applies padding.

**Comment by jonathanprozzi** [comment]
I think react-icons is a great call. One thing we can maybe do in our component library team meetings is to decide if there's a certain subset of react-icons we'd all want to use and align around that.

**Comment by earth2travis** [comment]
Blocked by: https://github.com/HausDAO/daohaus-monorepo/issues/36

**Comment by jordanlesich** [comment]
Figured it would be good to make a generic input that can be used anywhere instead. Then we can add it to the input as part of the InputLabel PR.

**Comment by jonathanprozzi** [comment]
That sounds good. Making the InputLabep able to take this generic in as children?

**Comment by jordanlesich** [comment]
I think it will compose tooltip, then render it if anything (string or component) is passed to the info prop. Similar to what's going on above, just through one more level. This way, InputLabel becomes more like a compound element that handles the label text, the required asterisk, and the tootltip based on props. 

It will help take some of the heavy lifting off FieldWrapper. It also allows us to compose that combo outside of field wrapper.

**Comment by jonathanprozzi** [comment]
Closed by #77
